pages/docs/guides/validate.md

```diff
@@ -0,0 +1,138 @@
+
+<Remember>
+Official parsers use <a href='https://github.com/asyncapi/spec-json-schemas/'>JSON Schema</a> created for AsyncAPI specification. JSON Schema is not enough to fully validate AsyncAPI documents. Learn more about <a href='https://github.com/asyncapi/spec-json-schemas#custom-validation-needs'>custom JSON Schemas validation needs</a>. Official JavaScript parser supports and validates these special needs.
+
+Take it into account if you're thinking about writing your own parser using official JSON Schema.
+</Remember>
+
+### Validation against best practices or company governance rules
+Now let's discuss options for validating against best practices or company governance rules, also known as **linting**. When various teams use AsyncAPI, you want to ensure they follow the same rules and are consistent across the organization. It is not enough to validate AsyncAPI documents against official specification rules. 
+
+<Remember>
+
+Let's discuss an example. While the `summary` property is optional in an AsyncAPI document, you could choose to require it for your organization. You would then implement a solution that enables you to enforce internal rules on AsyncAPI documents' providers.
+
+</Remember>
+
+One way this can be done is by using **Spectral**, an API linting tool which has a built-in [custom ruleset properties](https://meta.stoplight.io/docs/spectral/e5b9616d6d50c-custom-rulesets) with [AsyncAPI rules](https://meta.stoplight.io/docs/spectral/1e63ffd0220f3-async-api-rules) for the AsyncAPI specification. It also enables you to define company-specific rules that you can use internally.  
+
+To get started:
+1. Install [Spectral](https://meta.stoplight.io/docs/spectral/b8391e051b7d8-installation). 
+2. Create a file named `.spectral.yaml` to begin writing your API description and document rules. 
+    Example:
+    ```js
+    {
+      "formats": ["asyncapi2"],
+      "extends": "spectral:asyncapi",
+      "rules": {
+        // add your own rules here
+      }
+    }
+    ```
+
+3. Create and add your own custom ruleset:
+ ```js
+    {
+      "formats": ["asyncapi2"],
+      "extends": "spectral:asyncapi",
+      "rules": {
+        // add your own rules here
+        "valid-document-version": {
+          "message": "Version must match 2.x.x",
+          "severity": "hint",
+          "given": "$.info",
+          "then": [
+            {
+              "field": "version",
+              "function": "defined"
+            },
+            {
+              "field": "version",
+              "function": "pattern",
+              "functionOptions": {
+                "match": "^[0-9]+$"
+              }
+            }
+          ]
+        }
+      }
+    }
+  ```
+
+4. After setting up Spectral and creating custom rules following steps 1 - 3, validate your AsyncAPI document using this Spectral CLI command:
+
+```
+spectral lint asyncapi.yaml
+```
```

Comment:
```suggestion
    ```
    spectral lint asyncapi.yaml
    ```
```

without indentation it renders like it is not part of point 4
![Screenshot 2022-12-01 at 15 22 29](https://user-images.githubusercontent.com/6995927/205076998-6a69194a-d697-42ca-952b-b222821a3d59.png)